scripts/plotting/global_latlon_map.py

```diff
@@ -291,11 +296,16 @@ def global_latlon_map(adfobj):
 
                         # difference: each entry should be (lat, lon)
                         dseasons[s] = mseasons[s] - oseasons[s]
+                        
+                        # percent change
+                        #pseasons[s] = (abs(mseasons[s] - oseasons[s])/((mseasons[s] + oseasons[s])/2.0))*100.0 #absolute
```

Comment:
Could you remove this commented out line?

Reply:
Done